src/types/enum.types.ts

```diff
@@ -34,7 +34,7 @@ export enum VoteOptions {
 export enum EndPoint {
 	VOTE = "/vote",
 	ADD_PROPOSAL = "/addProposal",
-	ALL_DAO = "/allDao",
+	ALL_DAO = "/",
```

Comment:
Why???

Reply:
usually the first/starting page is assigned as / so I changed `allDao` it to /  as it will be the first page.
If /allDao makes more sense I can update it as well, what do you think ?

Reply:
Ok